.github/issuecomplete.yml

```diff
@@ -0,0 +1,27 @@
+# Configuration for Issue Check: https://github.com/stevenzeck/issue-check
+
+# The name of the label to apply when an issue does not have all tasks checked
+labelName: "status: needs-additional-info"
+
+# The color of the label in hex format (without #). Will use the label color configured
+# in Issues settings.
+labelColor:
+
+# The text of the comment to add to the issue in addition to the label
+commentText: >
+  Hello! It doesn't seem like we have quite enough information to send this to 
+  a human yet to help out. We would love if you could provide more details about 
+  your issue by following the template without modifying any of the pre-filled
+  text.
+
+# Whether or not to ensure all checkboxes are checked
+checkCheckboxes: true
+
+# Keywords to look for in the body of the issue
+keywords:
+  - Platform
+  - SDK version
+  - OS version
+  - Xcode version
+  - Debug logs
```

Comment:
will this effectively mark debug logs as required? they seem optional based on the copy in the bug report template, but we're checking for its checkbox.
would opening an issue without debug logs get an auto-reply asking for them?

Reply:
Opening an issue with "2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue" removed would generate an auto-reply saying the issue is incomplete and to follow the template. Generally what I've seen in issues across the SDK repos is that developers would delete the entire template and include fractions of the information

Reply:
I see, so as long as they don't delete, even if the check box isn't checked, they wouldn't get an auto-reply, right?

Reply:
There currently isn't a checkbox for debug logs, just if they read the mentioned links + updated Purchases SDK. As long as all checkboxes are checked and step 2 isn't deleted, they won't get an auto-reply. Once the template is correct, the bot will remove the `status: needs-additional-info` label

Reply:
oh right, I've no idea why I thought there was a checkbox for debug logs, my bad. Sounds great!